packages/mux-player/src/template.ts

```diff
@@ -108,23 +108,20 @@ export const content = (props: MuxTemplateProps) => html`
         : html``}
     </mux-video>
     ${isLiveOrDVR(props)
-      ? html`<button
+      ? html`<media-live-button
           slot="seek-live"
           part="${isLive(props) ? 'top' : 'bottom'} seek-live button"
           disabled="${props.inLiveWindow || !props.hasSrc || props.isDialogOpen}"
           aria-disabled="${(props.inLiveWindow || !props.hasSrc || props.isDialogOpen) && 'true'}"
-          in-live-window="${props.inLiveWindow}"
+          media-time-is-live="${props.inLiveWindow}"
           onclick="${function (this: HTMLButtonElement, evt: Event) {
```

Comment:
I don't think we need this onclick anymore, as it does it natively now: https://github.com/muxinc/media-chrome/blob/8c1e730b4baaa316f285d2be719451a1dea55c00/src/js/media-live-button.js#L59-L72